README.md

```diff
@@ -38,9 +38,11 @@ Here is an example of a fully functional Flatfile **Workbook** that we'll use in
 const Employees = new Sheet(
   'Employees',
   {
-    firstName: TextField({
+    firstName: LinkedField({
+      label: 'First Name',
       required: true,
-      description: 'Given name',
+      description: 'Give Name',
```

Comment:
Given name is a term for a person's "first name".  https://en.wikipedia.org/wiki/Given_name 

Possibly too confusing a concept for an introduction.